vhdbuilder/packer/packer_source.sh

```diff
@@ -65,6 +65,14 @@ copyPackerFiles() {
   CI_SYSLOG_WATCHER_SERVICE_DEST=/etc/systemd/system/ci-syslog-watcher.service
   CI_SYSLOG_WATCHER_SCRIPT_SRC=/home/packer/ci-syslog-watcher.sh
   CI_SYSLOG_WATCHER_SCRIPT_DEST=/usr/local/bin/ci-syslog-watcher.sh
+  AKS_LOGROTATE_SCRIPT_SRC=/home/packer/aks-logrotate.sh
+  AKS_LOGROTATE_SCRIPT_DEST=/usr/local/bin/aks-logrotate.sh
+  AKS_LOGROTATE_SERVICE_SRC=/home/packer/aks-logrotate.service
+  AKS_LOGROTATE_SERVICE_DEST=/etc/systemd/system/aks-logrotate.service
+  AKS_LOGROTATE_TIMER_SRC=/home/packer/aks-logrotate.timer
+  AKS_LOGROTATE_TIMER_DEST=/etc/systemd/system/aks-logrotate.timer
+  AKS_LOGROTATE_CONF_SRC=/home/packer/aks-rsyslog
+  AKS_LOGROTATE_CONF_DEST=/etc/logrotate.d/aks-rsyslog
```

Comment:
I suppose this automatically gets picked up by logrotate when the timer invokes it?

Reply:
yep yep, /etc/logrotate.conf has an include directive that brings in the dropins from /etc/logrotate.d/, which in this case will incluse aks-rsyslog which is overriding rsyslog for our purposes.